docs/geedocs/5.2.5/answer/7160006.html

```diff
@@ -136,6 +141,11 @@ <h5>
             <td>Symlinks are incorrectly world-writable files after installation from RPM or Deb</td>
             <td>Symlinks are now detected during the building of RPMs and Debs and are copied into the package correctly.</td>
           </tr>
+          <tr>
+             <td>1120</td>
+             <td>Installing fusion without server fails because of a missing vars file</td>
```

Comment:
These two new enhancements need to be documented:
https://github.com/google/earthenterprise/issues/1079
https://github.com/google/earthenterprise/issues/1068

See list of resolved issues for 5.2.5:
https://github.com/google/earthenterprise/issues?q=is%3Aissue+milestone%3A5.2.5+is%3Aclosed

Reply:
Add a reference to 1079, 1068 was mentioned already in the line about new publish options from the command line. Couldn't find anything else in the list of resolved issues that needed added.